content/learn/userguides/tertiary-analysis/tertiary-analysis.md

```diff
@@ -5,3 +5,7 @@ title: "Tertiary Analysis"
 ---
 
 ## Overview of Tertiary Analysis
+
+"Teriary analysis" is a term generally referring to analysis outside of the data portal. This may refer to downloading HCA data and using it in *de facto* software tools or interacting with a portal that communicates transparently with the Data Portal. These solutions are not built by the HCA DCP implementation team but are solutions we foster and support.
+
+Soon, we will turn on the Data Portal for initial use. During that time, we will begin to develop this space to grow into a collection of guides on how to use HCA data and perform analysis, as a data scientist, using software that is compatable with the DCP system.
```

Comment:
This second paragraph will have to be updated soon (because you refer to initial use of the portal); to make it more time-independent, you could change it to this sentence.
We are developing this space into a collection of guides describing how to use the HCA data and perform analysis, as a data scientist, using software that is compatible with the DCP system.